server/tso/allocator_manager.go

```diff
@@ -144,6 +139,9 @@ func (am *AllocatorManager) GetClusterDCLocations() (map[string][]uint64, error)
 		// The key will contain the member ID and the value is its dcLocation
 		serverPath := strings.Split(string(kv.Key), "/")
 		dcLocation := string(kv.Value)
+		if len(strings.TrimSpace(dcLocation)) == 0 {
```

Comment:
why need trim space?

Reply:
I'm not sure whether we could possibly get an empty value here from etcd, so I add this to make sure the value is valid as a `dc-location` string. If it's my overthinking, I will delete it though.